_extras/guide.md

```diff
@@ -1,6 +1,24 @@
 ---
 title: "Instructor Notes"
 ---
-FIXME
+
+## Episode 3: Monitor the training process
+When episode 3 is taught on a different day then episode 2, it is very useful to start with a recap of episode 2. This will help learners in the big exercise on creating a neural network.
+
+If learners did not download the data yet, they can also load the data directly from zenodo (instead of first downloading and saving):
```

Comment:
If that is the case, we should consider using this trick in the episode directly.

Reply:
We discussed that it's probably better to have the learners download the dataset in advance, and that this is 'trick' cannot be used for workshop taught on machines that do not have direct access to the internet. 
But we could include it as optional tip. Would a call-out box be a good place for this?

Reply:
I think a call out box is a good solution for this.